CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Added exponential backoff window module
 - Added parametric Galois LFSR module with optional whitening feature
+- `counter`: Added overflow latch
```

Comment:
the word "latch" could be misleading in this context. can we replace it with something like "maximum detection"?

Reply:
This has nothing to do with the maximum value, but rather the overflow bit of the counter remains set until the next clear. The parameter to control this is `LATCH_OVERFLOW`. Do you think this is misleading?

Reply:
I would not use the term `latch` in RTL design unless you actually refer to latches. we could name it "overflow detection" or similar.

Reply:
I suggest we call this "sticky overflow" instead, to distinguish it from the prior feature that already detected overflows. I have implemented that.